modules/apps/frontend-js/frontend-js-a11y-web/jest-setup.config.js

```diff
@@ -12,6 +12,6 @@
  * details.
  */
 
-import type {Result} from 'axe-core';
-declare const violations: Result[];
-export default violations;
+const sub = (string) => string;
+
+window.Liferay.Util.sub = sub;
```

Comment:
Is this necessary? I think when running liferay-npm-scripts this mock is already defined

Reply:
Yeah I thought the same when I was using it but it broke the tests because we need the `sub` to return at least the string value unlike what we do in `npm-scripts` which is a `jest.fn()`.